### DISCUSSION TITLE
NFT asset

### DISCUSSION THREAD

**Discussion by CFPro** [gochain-io/docs#102] [General]
I am trying to figure out how to find the actual NFT asset being transferred on the gochain.  Using the block explorer I can see funds being transferred, but nowhere do I see what asset was transferred, nor do I see a name of the NFT asset.  I am hoping somebody can help me. out. Here is the starting point:

https://explorer.gochain.io/addr/0x903470d09bcb2cA10E28e21a7095f696103E065d?addr_tab=transactions

Thanks in advance.

**Comment by treeder** [comment]
Did you transfer an NFT from that account? I don't see transactions that would be NFT transfers.

**Reply by CFPro** [reply]
Yes, under token transfer the very first entry for the value of - 5,607.149322961863 definitely transferred ownership of an NFT, here is the direct transaction hash 

https://explorer.gochain.io/tx/0x6c49b007986f494f059fb0c3b77a1d663aabaacbd5272e5047f7305995a0d062

**Reply by treeder** [reply]
That's an OMI token transfer, not an NFT.

**Reply by CFPro** [reply]
Ah where can I locate the NFT transfer ?

**Reply by CFPro** [reply]
Is there a correlation between the token transfer and the NFT transfer?

**Reply by treeder** [reply]
None whatsoever. How did you transfer the NFT? You should have a transaction ID for the transfer.

**Reply by CFPro** [reply]
That is too bad.  I will hunt down that transaction ID and explore further.

**Reply by treeder** [reply]
Find it?

**Reply by CFPro** [reply]
They stated that they are working on the ability to show my NFTs on the blockchain.
